runtime/stdlib/rlp.go

```diff
@@ -94,16 +94,42 @@ func RLPBuiltInFunctions(impls RLPBuiltinImpls) StandardLibraryFunctions {
 func DefaultRLPBuiltinImpls() RLPBuiltinImpls {
 	return RLPBuiltinImpls{
 		RLPDecodeString: func(invocation interpreter.Invocation) interpreter.Value {
-			// input := invocation.Arguments[0].(*interpreter.ArrayValue)
-			// output, err := rlp.DecodeString(input, 0)
-			// if err != nil {
-			// 	panic(err)
-			// }
-			// return output
-			panic(fmt.Errorf("cannot rlp decode string"))
+			input := invocation.Arguments[0].(*interpreter.ArrayValue)
+
+			convertedInput, err := interpreter.ByteArrayValueToByteSlice(input)
+			if err != nil {
+				panic(err) // TODO should I panic this way?
```

Comment:
This is correct: If the input is invalid, or below if the decoding fails, we want the whole program execution to abort, which we do using `panic`s in the interpreter 👍